README.md

```diff
@@ -1,6 +1,130 @@
 # application-aware-quota
 Empower users to customize and manage resource quotas per component in Kubernetes, utilizing the scheduling gates feature.
 
+## Motivation
+In a multi-tenant cluster where multiple users or teams share a Kubernetes cluster with a fixed number of nodes, 
```

Comment:
Well the reason i added this is because i saw it in Kubernetes [docs](https://kubernetes.io/docs/concepts/policy/resource-quotas/):
"When several users or teams share a cluster with a fixed number of nodes, there is a concern that one team could use more than its fair share of resources."

But i guess you are right it doesn't have to be a fixed number of node i'll remove it